packages/server/src/http.ts

```diff
@@ -61,7 +61,15 @@ export function createServer(app: Application, options: ServerOptions): Resource
         try {
           yield;
         } finally {
-          server.close();
+          let { future, produce } = createFuture<void>();
```

Comment:
Let's use `resolve()` and `reject()` in future instead since they're simpler. Also, we may deprecate `produce()` at some point.

Reply:
In v0 the old version of effection, there is no `resolve/reject`